frontend/src/v5/store/containers/containers.redux.ts

```diff
@@ -189,6 +192,26 @@ export const setContainerStatus = (state = INITIAL_STATE, {
 	},
 });
 
+export const containerProcessingSuccess = (state = INITIAL_STATE, {
```

Comment:
Lets keep this for now, but I think a more general updateContainerSuccess function could replace  
_fetchContainerStatsSuccess,  fetchContainerViewsSuccess , fetchContainerSettingsSuccess,  updateContainerSettingsSuccess  and containerProcessingSuccess, setFavouriteSuccess_ functions